### PR TITLE
Add env variables to connect to staging URLs

### PR DESCRIPTION
**Pull request by aashishgurung**
## Purpose
Add env variables to connect to staging URLs via godotenv pacakge.

## Description
Endpoint file is updated to get API and Vault URLs from `.env` if present. Replaced `internal.API` with `internal.GetEnv().ApiUrl`. Used [godotenv](https://github.com/joho/godotenv) package to read env files.

### CODE REVIEW

File: internal/endpoint.go [hunk 1]
```diff
@@ -1,8 +1,50 @@
 package internal
 
-type Endpoint string
+import (
+	"os"
 
-const (
-	API   Endpoint = "https://api.omise.co"
-	Vault          = "https://vault.omise.co"
+	"github.com/joho/godotenv"
```
Comment (danfowler) [inline internal/endpoint.go:6]: I wouldn't include `joho/godotenv` here, but rather leave it to the application.

File: operations/recipient.go [hunk 2]
```diff
@@ -235,17 +229,15 @@ func (req *DestroyRecipient) Describe() *internal.Description {
 //	if e := client.Do(&schds, &list); e != nil {
 //		panic(e)
 //	}
-//
-//	fmt.Println("# of schedules made in the last hour:", len(schds.Data))
-//
+fmt.Println("# of schedules made in the last hour:", len(schds.Data))
```
Comment (ajzkk) [inline operations/recipient.go:232]: Code is failing due to this line modification.

**Comment by ajzkk** [comment]
It will also need to change endpoint in `token.go`

**Comment by danfowler** [comment]
@aashishgurung close?

**Pull request closed without merging**